src/GraphQL/Types/Scalars/BooleanGraphType.cs

```diff
@@ -9,15 +9,26 @@ namespace GraphQL.Types
     public class BooleanGraphType : ScalarGraphType
     {
         /// <inheritdoc/>
-        public override object ParseLiteral(IValue value) => ((value as BooleanValue)?.Value).Boxed();
+        public override object ParseLiteral(IValue value) => value switch
+        {
+            BooleanValue b => b.Value.Boxed(),
+            NullValue _ => null,
+            _ => null
```

Comment:
Why not `ThrowLiteralConversionError` ?

Reply:
Bug; throwing is necessary.  I will be writing tests also.  Good to have you reviewing everything also to help catch these things!

Reply:
> I will be writing tests also

Tests will save us.